src/Fantomas.Tests/UtilsTests.fs

```diff
@@ -111,4 +113,67 @@ SetupTesting.generateSetupScript __SOURCE_DIRECTORY__
 #load "__setup__.fsx"
 #endif
 """
-    |> mergeAndCompare a b
+        |> mergeAndCompare a b
+
+module List =
+    [<Test>]
+    let ``When input is empty`` () =
+        let property (p: bool) : bool =
+            let before, after = List.partitionWhile (fun _ _ -> p) []
+            before = [] && after = []
+
+        Check.QuickThrowOnFailure(property true)
+        Check.QuickThrowOnFailure(property false)
```

Comment:
Guessing this isn't quite idiomatic use of FsCheck, but I couldn't find a simple way to say "use true and false". I could do this:

```fsharp
Check.QuickThrowOnFailure property
```

But then it runs multiple times with the same Boolean input. I could also do:

```fsharp
[true;false]
|> List.iter (fun p -> Check.QuickThrowOnFailure(property p)
```

But not sure if that's any better.

Reply:
@Smaug123 or @jindraivanek will have to comment on this one, I haven't really used FsCheck in a while.